build/js/movie_details.js

```diff
@@ -41,9 +41,11 @@ function buildDetail(movie) {
   const genre = getGenres(movie);
   const releaseInfo = getReleaseInfo(dateInfo);
 
+  // <a href="javascript:window.history.back();">go back</a>
+
   const html = `
     <div class="big_card">
-      <div id='goback' class='goback'>&#x2190; Go Back</div>
+      <div id='goback' class='goback'><a href="javascript:window.history.back();">&#x2190; Go Back</a></div>
```

Comment:
This approach works for the most part but is typically not advised.  If someone were to navigate to this page directly, for example, where would `back` take them?  In that instance, what you want them to do is head to the `index.html` page, so I would update this code to simply link them to that page all the time.
```suggestion
      <div id='goback' class='goback'><a href="/index.html">&#x2190; Go Back</a></div>
```

Reply:
The reason I did it that way was because if they are on say year 2017 and "go back" it would take them to 2019 which isn't where they were. I can see that being an issue not having a "back" to go to.